tests/test_javascript.py

```diff
@@ -47,3 +48,41 @@ def get_javascript_name():
     assert (
         first_js != second_js
     ), "add_js should add to the previous script in the catalog."
+
+
+def test_startup_dest(pdf_file_writer):
```

Comment:
The `/OpenAction` property doesn't explicitly have to do with JS, so I'd vote to move this into `test_writer.py`.